apps/dashboard/src/hooks/useOnchainGroups.ts

```diff
@@ -21,9 +26,11 @@ export default function useOnchainGroups(): ReturnParameters {
                         (group) => getAddress(group.admin) === getAddress(admin)
                     )
                     .map((group) => {
+                        const groupName = formatGroupName(group.id)
+
                         return {
-                            name: group.id, // TODO: convert to string
-                            description: `${group.id} on-chain group`,
+                            name: groupName,
+                            description: `${groupName} on-chain group`,
```

Comment:
That description doesn't make much sense to me, as it's always the same. But that should be part of another PR I guess

Reply:
Yea, I didn't like it either. Not that its static, but its not properly formatted. It was probably added to keep the UI component same as that of off-chain group where we actually have a description.
We can address it later.